opennms-container/horizon/container-fs/confd/conf.d/slack.properties.toml

```diff
@@ -0,0 +1,10 @@
+[template]
+src = "slack.properties.tpl"
+dest = "/opt/opennms/etc/opennms.properties.d/_confd.slack.properties"
+keys = [
+   "opennms/notifd/slack/channel",
+   "opennms/notifd/slack/userName",
+   "opennms/notifd/slack/iconEmoji",
+   "opennms/notifd/slack/iconURL",
+   "opennms/notifd/slack/useSystemProxy"
```

Comment:
We can simplify this to just include the base path for slack
```
keys = [
   "opennms/notifd/slack"
]
```

Reply:
I just looked at some of the other horizon templates and noticed they are using the convention of listing the individual keys, so if you want to continue that in your change you can to remain consistent. Up to you.

Reply:
Let's make it less redundant.